File: premium-calculator-api/Models/PremiumRequest.cs
namespace premium_calculator_api.Models
{
    public class PremiumRequest
    {
        public string Name { get; set; }
        public int AgeNextBirthday { get; set; }
        public string Dob { get; set; }  // MM/YYYY
        public int OccupationId { get; set; }
        public decimal DeathSumInsured { get; set; }
    }
}

File: premium-calculator-UI/premium-calculator-ui/src/app/Models/PremiumRequest.ts
export interface PremiumRequest {
  name: string;
  ageNextBirthday: number;
  dob: string; // MM/YYYY
  occupationId: number;
  deathSumInsured: number;
}

export interface PremiumResponse {
  monthlyPremium: number;
}

export interface Occupation {
  id: number;
  name: string;
  rating: string;
}
